ci/travis/install-osx.sh

```diff
@@ -32,7 +32,7 @@ python --version
 
 # Build NuPIC
 cd ${TRAVIS_BUILD_DIR}
-python setup.py install --user
+ARCHFLAGS="-arch x86_64" python setup.py install --user
```

Comment:
Maybe this is separate from this PR but I don't set this env variable and have no problem on OS X. Why do we think it is necessary?